packages/protoc-gen-connect-query/package.json

```diff
@@ -21,10 +21,12 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "pnpm clean && tsc --project tsconfig.build.json",
-    "format": "prettier . --write --ignore-path ./.eslintignore && eslint . --fix && license-header"
+    "format": "prettier . --write --ignore-path ./.eslintignore && eslint . --fix && license-header",
+    "generate": "rm -rf snapshots/gen && cd snapshots && buf generate --path eliza.proto"
```

Comment:
Since we're using the unmodified eliza.proto, `buf generate buf.build/connectrpc/eliza:8b8b971d6fde4dc8ba5d96f9fda7d53c` means we could remove the copy. We end up with more directories though.

Leaving it up to you.